File: Python/String-1/make_abba.py
"""
Dada 2 Strings ("a" e "b"), retorne outra no fomato "abba".
Ex.:(('Hi', 'Bye') → 'HiByeByeHi'; ('What', 'Up') → 'WhatUpUpWhat'; ('Yo', 'Alice') → 'YoAliceAliceYo').
"""


def make_abba(a, b):
    return a + 2*b + a


print(make_abba("Hi", "bye"))

File: Java/String-1/src/makeAbba/MakeAbba.java
package makeAbba;

public class MakeAbba {
    //Dada as Strings a e b então retorne abba. (Ex.: ("hi","bye") -> "hibyebyehi").

    public String makeAbba(String a, String b) {
        return a+b+b+a;
    }
}
